File: RandomUserStats/Models/UserStats.cs
using System.Collections.Generic;

namespace RandomUserStats.Models
{
	public class UserStats
	{
		public double FemaleToMaleRatio { get; set; }
		public double AThroughMFirstNameRatio { get; set; }
		public double AThroughMLastNameRatio { get; set; }
		public List<StatePopulationPercentage> MostPopulousStates { get; set; }
		public AgeRangePercentages AgeRangePercentages { get; set; }
		public Dictionary<string, int> PreferredTitleStatistics { get; set;}
	}
}

File: RandomUserStats/ClientApp/src/app/user-stats.ts
export class UserStats {
    femaleToMaleRatio: number;
    aThroughMFirstNameRatio: number;
    aThroughMLastNameRatio: number;
    mostPopulousStates: StatePopulationPercentage[];
    ageRangePercentages: AgeRangePercentages;
}

export class StatePopulationPercentage {
    state: string;
    percentageOfTotalPopulation: number;
    femalePercent: number;
    malePercent: number;
}

export class AgeRangePercentages {
    zeroThroughTwenty: number;
    twentyOneThroughForty: number;
    fortyOneThroughSixty: number;
    sixtyOneThroughEighty: number;
    eightyOneThroughOneHundred: number;
    overOneHundred: number;
}
